website/docs/0. intro.md

```diff
@@ -1,3 +1,6 @@
+---
+slug: /
+---
```

Comment:
This block is visible from the main readme.

Reply:
ach OK had to do some shuffling to keep the slug out (pure markdown page without frontmatter included by frontmatter version.)